[INFO] 2025/06/30 04:07:28: Checking all possible Updates ......
[INFO] 2025/06/30 04:07:28: <Zerojudge> : Start fetching submissions ......
[INFO] 2025/06/30 04:07:28: <UVa> : Start fetching submissions ......
[INFO] 2025/06/30 04:07:28: <Kattis> : Start fetching submissions ......
[INFO] 2025/06/30 04:07:28: <TOJ> : Start fetching submissions ......
[INFO] 2025/06/30 04:07:28: <AtCoder> : Start fetching submissions ......
[INFO] 2025/06/30 04:07:28: <CodeForces> : Start fetching submissions ......
[INFO] 2025/06/30 04:07:29: <CodeForces> : Submissions fetching COMPLETE  :) ( 243 subs )
[INFO] 2025/06/30 04:07:29: <AtCoder> : Submissions fetching COMPLETE  :) ( 90 subs )
[INFO] 2025/06/30 04:07:32: <UVa> : Submissions fetching COMPLETE  :) ( 499 subs )
[WARNING] 2025/06/30 04:07:34: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f4b8eb99310>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/d1bc406954e5efc0550f44696029b150/elements
[WARNING] 2025/06/30 04:07:34: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f4b765927e0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/d1bc406954e5efc0550f44696029b150/elements
[WARNING] 2025/06/30 04:07:34: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f4b76592a20>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/d1bc406954e5efc0550f44696029b150/elements
[ERROR] 2025/06/30 04:07:34: <TOJ> : Unable to fetch submissions :(
